packages/design-tokens/sass/color-vars.scss

```diff
@@ -56,6 +56,62 @@ $kz-var-color-wisteria-800-rgb-params: var(
   55,
   74
 );
+$kz-var-color-purple-100: var(--kz-var-color-purple-100, #f0f1f4);
```

Comment:
So these variables are only here to help with the transition. When the breaking change is made the <thing>-var.scss files will be deleted.